planemo/workflow_lint.py

```diff
@@ -138,9 +145,22 @@ def lint_workflow_artifacts_on_paths(
 def _lint_workflow_artifacts_on_path(
     lint_context: WorkflowLintContext, path: str, lint_args: Dict[str, Union[str, List[str]]]
 ) -> None:
+    iwc_grade = lint_args["iwc_grade"] == "True"
+    if iwc_grade:
+        if not os.path.isdir(path):
```

Comment:
Can we instead just look at the directory the workflow is in ? I think that's more user-friendly.

Reply:
So, if you run with `--iwc` on a workflow file, it checks the files are in the directory?

Reply:
If the given path is a workflow file, we look at the containing directory, if it's a directory then we use that. There's other places in planemo where we do the same.